cmd/meroxa/global/client.go

```diff
@@ -98,8 +98,8 @@ func GetCLIUserInfo() (actor, actorUUID string, err error) {
 }
 
 func GetUserToken() (accessToken, refreshToken string, err error) {
```

Comment:
@terrancej what do you think about adding compatibility with those that still have these without `MEROXA_` prefix? Something similar to what we do in https://github.com/meroxa/cli/blob/master/cmd/meroxa/global/config.go#L77-L79.